Formula/texmath.rb

```diff
@@ -7,7 +7,7 @@ class Texmath < Formula
   homepage "https://johnmacfarlane.net/texmath.html"
   url "https://hackage.haskell.org/package/texmath-0.12.0.2/texmath-0.12.0.2.tar.gz"
   sha256 "2fec285a2266e56bba17914c122045f31b38de3efcd202dcf32a4f8b830bd184"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
I think it's `GPL-2.0-or-later`. If you look in the `src` some of the files have this header:

> Copyright (C) 2009-2012 John MacFarlane <jgm@berkeley.edu>
>
> This program is free software; you can redistribute it and/or modify
> it under the terms of the GNU General Public License as published by
> the Free Software Foundation; either version 2 of the License, or
> (at your option) any later version.
>
> This program is distributed in the hope that it will be useful,
> but WITHOUT ANY WARRANTY; without even the implied warranty of
> MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> GNU General Public License for more details.
>
> You should have received a copy of the GNU General Public License
> along with this program; if not, write to the Free Software
> Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA

Reply:
The major reference is in here, https://hackage.haskell.org/package/texmath-0.12.0.2

Reply:
I'm not sure what to do here. There seem to be a few conflicts where the haskell site differs from the source code.

@jonchang can you advise here?

Reply:
this kind of reminds me the similar situation that we had with Github license declarations. We probably should use the source code as license, but we also need to actively help the upstream repo to conform with SPDX license stuff.

Reply:
I can definitely break down this PR into small chunks of mergeable PRs and iterate on it. :)